test/ExpressiveInstallerTest/RoutersTest.php

```diff
@@ -93,9 +106,27 @@ public function testRouter(
         );
 
         // Test home page
-        $setupRoutes = (strpos($copyFilesKey, 'minimal') !== 0);
+        $setupRoutes = strpos($copyFilesKey, 'minimal') !== 0;
         $response = $this->getAppResponse('/', $setupRoutes);
         $this->assertEquals($expectedResponseStatusCode, $response->getStatusCode());
+
+        /** @var Application $app */
+        $app = $container->get(Application::class);
+        $this->assertCount(count($expectedRoutes), $app->getRoutes());
+        foreach ($app->getRoutes() as $route) {
+            foreach ($expectedRoutes as $expectedRoute) {
+                if ($expectedRoute['name'] === $route->getName()) {
+                    $this->assertEquals($expectedRoute['path'], $route->getPath());
+                    // todo: not sure how we can test middleware?
+                    // $this->assertEquals($expectedRoute['middleware'], $route->getMiddleware());
```

Comment:
How do we know then if correct middleware is executed? As I understand correctly, with your suggested changes we will check just response codes for each route.
It's hard to check what middleware is bind for specific route, what I wanted to check here.
Maybe it's not important and what you've suggested will be sufficient?